config.lua

```diff
@@ -12,6 +12,7 @@ Config.LockPickDoorEvent = function() -- This function is called when a player a
 end
 
 -- Carjack Settings
+Config.enablecarjack = true
```

Comment:
`EnableCarJacking` would maybe be more consistent with the other config items?
 
A comment to describe it also woudln't go a miss :)

Reply:
Maybe `CarJackEnable`? 
That way all of the `CarJack` config items can follow the same pattern, i.e:
`Config.CarJackEnable`
`Config.CarJackChance`
`Config.CarJackCooldown`
etc..

Reply:
I made the suggested edits.